### ISSUE TITLE
`docker run` fails to find pkgxdev/pkgx image

### ISSUE THREAD

**Issue by rytswd** [pkgxdev/pkgx#769]
Following the documentation https://docs.pkgx.sh/run-anywhere/docker, I couldn't pull the docker image -- perhaps it's set as private?

```console
❯ docker run -it pkgxdev/pkgx
Unable to find image 'pkgxdev/pkgx:latest' locally
docker: Error response from daemon: pull access denied for pkgxdev/pkgx, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.
```

**Comment by mxcl** [comment]
yes sorry, we are working to get this live. /cc @Xercesblu3

**Comment by mxcl** [comment]
docker image is now live. It is a little less functional than I'd like (no shell integration) but that will be fixed for the 1.0.1 release.
